scripts/validate_templates.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+# TODO: fix /libraries
```

Comment:
`fix` should be _remove_ libraries. I think I added some code in the generation script to only look in certain places to scan for translatable strings in the first place, since there is so much  code that is unused by the front-end.

Reply:
I strongly agree that there should only be a single source of truth for the list of templates and continued to use the mechanism that @hornc put in place, but would **strongly** prefer that dead code/templates be deleted/removed so that the list valid templates is just everything in the correct directory hierarchy. Dead code wastes a lot more effort than just internationalization/translation.